src/workHome/CalcWagServlet.java

```diff
@@ -0,0 +1,37 @@
+package workHome;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/calcwag")
+public class CalcWagServlet extends HttpServlet {
+    private Calculator calculator = new Calculator();
```

Comment:
Servlet jest raczej bezstanowy - nie powienien mieć takich obiektów które zmienają swój stan za każdym razem kiedy ktoś wywołuje jakieś żądanie. Ten obiekt powinien być tworzony w chwilii kiedy będzie on potrzebny. U Ciebie jest to sekcja `else`.